### PR TITLE
vim-patch:6e649224926b

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/6e649224926bbc1df6a4fdfa7a96b4acb1f8bee0

skip doc/autocmd.txt (needs https://github.com/neovim/neovim/pull/15840)

### CODE REVIEW

**Comment by zeertzjq** [comment]
I think you meant doc/autocmd.txt

**Comment by zeertzjq** [comment]
Maybe you forgot the change in doc/intro.txt

**Comment by clason** [comment]
Weird, that should have been part of the scripted port. 🤷🏻 

Now it's in, thanks for pointing it out!

**Review by justinmk — APPROVED** [review]

File: runtime/doc/eval.txt [hunk 1]
```diff
@@ -7007,7 +7007,8 @@ nr2char({expr} [, {utf8}])				*nr2char()*
 			nr2char(32)		returns " "
 <		Example for "utf-8": >
 			nr2char(300)		returns I with bow character
-<		UTF-8 encoding is always used, {utf8} option has no effect,
+<		When {utf8} is TRUE, always return UTF-8 characters.
+		UTF-8 encoding is always used, {utf8} option has no effect,
```
Comment (seandewar) [inline runtime/doc/eval.txt:7011]: Ditto

```suggestion
<		UTF-8 encoding is always used, {utf8} option has no effect,
```

**Comment by zeertzjq** [comment]
I think this is why the vim-patch.sh didn't include it:
<https://github.com/neovim/neovim/blob/0c2a7aa5f75502364a884eb25787d7774033f817/scripts/vim-patch.sh#L197>

Probably because there are a lot of differences between and Neovim's intro.txt

File: runtime/doc/eval.txt [hunk 2]
```diff
@@ -6345,8 +6345,8 @@ list2str({list} [, {utf8}])				*list2str()*
 <		|str2list()| does the opposite.
 
 		When {utf8} is omitted or zero, the current 'encoding' is used.
-		With {utf8} is 1, always return utf-8 characters.
-		With utf-8 composing characters work as expected: >
+		When {utf8} is TRUE, always return UTF-8 characters.
+		With UTF-8 composing characters work as expected: >
```
Comment (seandewar) [inline runtime/doc/eval.txt:6349]: `{utf-8}` is ignored in Nvim.

File: runtime/doc/eval.txt [hunk 3]
```diff
@@ -8973,8 +8974,8 @@ str2list({string} [, {utf8}])				*str2list()*
 <		|list2str()| does the opposite.
 
 		When {utf8} is omitted or zero, the current 'encoding' is used.
-		With {utf8} set to TRUE, always treat the String as utf-8
-		characters.  With utf-8 composing characters are handled
+		When {utf8} is TRUE, always treat the String as UTF-8
+		characters.  With UTF-8 composing characters are handled
```
Comment (seandewar) [inline runtime/doc/eval.txt:8978]: Ditto

**Comment by clason** [comment]
I assumed it was skipped, but the change seems innocuous, so it's OK to include it anyway.

**Comment by zeertzjq** [comment]
> I assumed it was skipped, but the change seems innocuous, so it's OK to include it anyway.

Actually I proposed it. <https://github.com/vim/vim/issues/8918>

**Review by seandewar — COMMENTED** [review]
Sorry about the late review! Been a busy day!

Btw, this seems to be missing the `nginx.vim` files?

File: runtime/doc/editing.txt [hunk 4]
```diff
@@ -1183,7 +1183,8 @@ If you want to always use ":confirm", set the 'confirm' option.
 			|:diffsplit|, |:diffpatch|, |:pedit|, |:redir|,
 			|:source|, |:update|, |:visual|, |:vsplit|,
 			and |:qall| if 'confirm' is set.
-			{only in Win32 GUI}
+			{only in Win32 GUI, in console `browse edit` works
+			if the FileExplorer autocommand group exists}
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/editing.txt:1187]: Maybe, but it was there before? (I try not to combine vim-patch changes with my own.)

**Comment by clason** [comment]
Ah, I think I forgot to manually `git add` the `nginx.vim` file 🤦🏻  The diff was so short, it looked like a change, not a new file... (Not at my other machine where I opened the PR, so I can't verify.)

I'll open a fixup PR tonight. Sorry for jumping the gun!

File: runtime/doc/autocmd.txt [hunk 5]
```diff
@@ -1232,6 +1232,13 @@ Example: >
 This prevents having the autocommands defined twice (e.g., after sourcing the
 vimrc file again).
 
+						*FileExplorer*
```
[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/doc/autocmd.txt:1235]: Not sure it has any special behaviour in Nvim though.

E.g: Vim has https://github.com/vim/vim/blob/7f2dd1e90c1d4a30c791fae20014594641769a1e/src/ex_cmds.c#L2547 and https://github.com/vim/vim/blob/08d7b1c82866a61b61a55e55b6c190dba04e54ea/src/ex_docmd.c#L6523 but I don't see anything in Nvim like this 🤷

**Pull request merged**